url-parameters/config_reference.json

```diff
@@ -35,7 +35,8 @@
                     "mkt_tok",
                     "hmb_campaign",
                     "hmb_source",
-                    "hmb_medium"
+                    "hmb_medium",
+                    "bad[.*]"
```

Comment:
can we make it simpler? I don't think all regex implementations are "full" (i.e. support all special characters). How about "bad." and then check if it doesn't affect "badx" param

Reply:
Uh oh, or `rege.` and "regex" 👀